packages/botonic-react/src/webchat-app.jsx

```diff
@@ -175,7 +176,7 @@ export class WebchatApp {
       visibility,
       ...webchatOptions
     } = optionsAtRuntime
-    theme = { ...this.theme, ...theme }
+    theme = merge(this.theme, theme)
```

Comment:
it would be nice to have a unit test for this.
if you don't have time, you can create an empty one and mark it with test.skip so that it's highlighted

Reply:
just added a test for this 👍